jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(30000)
```

Comment:
this can go to config https://jestjs.io/docs/next/configuration#testtimeout-number